doc/api/errors.md

```diff
@@ -920,6 +920,11 @@ An invalid HTTP/2 header value was specified.
 An invalid HTTP informational status code has been specified. Informational
 status codes must be an integer between `100` and `199` (inclusive).
 
+<a id="ERR_HTTP2_INVALID_ORIGIN"></a>
+### ERR_HTTP2_INVALID_ORIGIN
+
+HTTP/2 ORIGIN frames require a valid origin.
```

Comment:
The format is inconsistent through the docs, but `` `ORIGIN` frames `` seems more readable?